docs/contents/releases.md

```diff
@@ -1,128 +1,159 @@
+controller -- the creation of a new SemVer Git tag on the source code
+repository will trigger automatic building and publishing of a Helm Chart with
+an artifact tag including the SemVer version. For example, a Git tag of
+`v1.2.6` on the [github.com/aws-controllers-k8s/s3-controller][s3-ctrl]
+repository means a Helm chart with a tag `s3-v1.2.6` would be published to the
+[aws-controllers-k8s/chart][ecr-ack-chart] ECR repository.
 
-## PROPOSED
+[ecr-ack-chart]: https://gallery.ecr.aws/aws-controllers-k8s/chart
 
-The `PROPOSEDD` status indicates that someone has expressed interest in
-supporting said service in ACK.
+### A Word About Dependencies
 
-There will be a Github Issue for tracking the build of the ACK service
-controller for the service.
+Each service-specific ACK controller -- e.g. the ElastiCache ACK controller --
+depends on a specific version of the ACK common runtime. This dependency is
+specified in the controller's `go.mod` file.
 
-The GitHub Issue **WILL NOT** be associated with a GitHub Milestone.
+The ACK code generator that produces Go code for service controllers depends on
+a specific version of the ACK common runtime.
 
-## PLANNED
+!!! note "dependency between the code generator and common runtime"
+    The ACK code generator depends on the ACK common runtime in a unique way:
+    the Go code that the ACK code generator *produces* adheres to a specific
+    version of the ACK common runtime. Even though no Go code in the ACK code
+    generator actually imports the ACK common runtime, this dependency exists
+    because the Go code produced by the templates inside the code generator imports
+    the ACK common runtime. In order to make this Go code dependency more strict,
+    we have a test package inside the ACK code generator that imports the ACK
+    common runtime. In this way, we're able to include a version-specific
+    dependency line in the ACK code generator's `go.mod` file, thereby allowing
+    Go's module infrastructure to pin the dependency between the code generator and
+    the common runtime.
 
-The `PLANNED` status indicates that **the AWS service is on our radar** for
-building into ACK.
+## Stable Helm Charts
 
-There will be a Github Issue for tracking the build of the ACK service
-controller for the service.
+We [recommend][recommend-helm] using Helm to install an ACK service controller.
 
-The GitHub Issue **WILL BE** associated with a GitHub Milestone indicating the
-target date for the `DEVELOPER PREVIEW` release of that service's controller.
+[recommend-helm]: https://aws-controllers-k8s.github.io/community/user-docs/install/#helm-recommended
 
-## BUILD
+If you want to only install ACK service controllers that have been determined
+to be stable by the ACK contributor team and the associated AWS service team,
+you may choose to install the Helm chart for an ACK service controller that has
+been tagged with the `-stable` suffix -- e.g. `elasticache-stable`.
```

Comment:
In the current proposal, each release may have a stable release or not

```
s3-v1.1.0
s3-v1.2.0, s3-v1.2.0-stable
s3-v.1.3.0
s3-v1.4.0
s3-v1.5.0, v1.5.0-stable
s3-v2.1.0, v2.1.0-stable
```
Is the above interpretation correct? or there will be a stable version for each release?
It will be easier if the user does not have to specifically look at which versions have been marked as stable. Also, production use-case should not be updating to 1.3/1.4, what is the purpose of releases with such artifacts?

Instead what if ACK publishes to a separate the repository for anything which should not be used in production. It can have nightly/weekly release cadence according to the service and release version would have a suffix with a date or short git SHA. It would look something like:

Preview/Nightly repo
```
s3-abcdef
s3-defghi
s3-ghiijk
.
.
s3-uvwxyz
```

Release/stable repo
```
s3-v1.1.0 // cut a release tag v1.1.0 on commit s3-abcdef, 
s3-v1.2.0 // cut a release tag v1.2.0 on commit s3-ghiijk
s3-v.1.3.0 // cut a release tag v1.3.0 on commit s3-uvwxyz
```

A release is always with a semver, stable and supported